### PR TITLE
Fix plugin example tests with `4.0.1-rc.1`

### LINKED ISSUE #6133: Errors at the tests of `web3-plugin-example` with `4.0.1-rc.1`
#### Steps to reproduce the behavior
run:
```bash
cd tools/web3-plugin-example/
yarn test
```
You will see those logged:
```bash
yarn run v1.22.19
$ jest --config=./test/unit/jest.config.js
 PASS  test/unit/custom_rpc_methods.test.ts
 FAIL  test/unit/contract_method_wrappers.test.ts
  ● CustomRpcMethodsPlugin Tests › CustomRpcMethodsPlugin methods tests › should call contractMethodWrappersPlugin.getFormattedBalance with expected RPC object

    expect(jest.fn()).toHaveBeenCalledWith(...expected)

    - Expected
    + Received

    @@ -1,10 +1,9 @@
      Object {
        "method": "eth_call",
        "params": Array [
          Object {
    -       "data": "0x70a082310000000000000000000000008da5e39ec14b57fb9bcd9aa2b4500e909119795d",
            "input": "0x70a082310000000000000000000000008da5e39ec14b57fb9bcd9aa2b4500e909119795d",
            "to": "0xdAC17F958D2ee523a2206206994597C13D831ec7",
          },
          "latest",
        ],,

    Number of calls: 1

      65 |                              DEFAULT_RETURN_FORMAT,
      66 |                      );
    > 67 |                      expect(requestManagerSendSpy).toHaveBeenCalledWith({
         |                                                    ^
      68 |                              method: 'eth_call',
      69 |                              params: [
      70 |                                      {

      at test/unit/contract_method_wrappers.test.ts:67:34
      at fulfilled (test/unit/contract_method_wrappers.test.ts:5:58)

  ● CustomRpcMethodsPlugin Tests › CustomRpcMethodsPlugin methods tests › should call CustomRpcMethodsPlugin.customRpcMethodWithParameters with expected RPC object

    expect(jest.fn()).toHaveBeenCalledWith(...expected)

    Expected: {"method": "eth_sendTransaction", "params": [{"data": "0xa9059cbb0000000000000000000000004f641def1e7845caab95ac717c80416082430d0d000000000000000000000000000000000000000000000000000000000000002a", "from": "0x8da5e39ec14b57fb9bcd9aa2b4500e909119795d", "gasPrice": "0x1ca14bd70", "input": "0xa9059cbb0000000000000000000000004f641def1e7845caab95ac717c80416082430d0d000000000000000000000000000000000000000000000000000000000000002a", "maxFeePerGas": undefined, "maxPriorityFeePerGas": undefined, "to": "0xdAC17F958D2ee523a2206206994597C13D831ec7"}]}
    Received
           1
              Object {
            -   "method": "eth_sendTransaction",
            +   "method": "eth_call",
                "params": Array [
                  Object {
            -       "data": "0xa9059cbb0000000000000000000000004f641def1e7845caab95ac717c80416082430d0d000000000000000000000000000000000000000000000000000000000000002a",
            -       "from": "0x8da5e39ec14b57fb9bcd9aa2b4500e909119795d",
            -       "gasPrice": "0x1ca14bd70",
            -       "input": "0xa9059cbb0000000000000000000000004f641def1e7845caab95ac717c80416082430d0d000000000000000000000000000000000000000000000000000000000000002a",
            -       "maxFeePerGas": undefined,
            -       "maxPriorityFeePerGas": undefined,
            +       "input": "0x70a082310000000000000000000000008da5e39ec14b57fb9bcd9aa2b4500e909119795d",
                    "to": "0xdAC17F958D2ee523a2206206994597C13D831ec7",
                  },
            +     "latest",
                ],
              },
           2
              Object {
            -   "method": "eth_sendTransaction",
            -   "params": Array [
            -     Object {
            -       "data": "0xa9059cbb0000000000000000000000004f641def1e7845caab95ac717c80416082430d0d000000000000000000000000000000000000000000000000000000000000002a",
            -       "from": "0x8da5e39ec14b57fb9bcd9aa2b4500e909119795d",
            -       "gasPrice": "0x1ca14bd70",
            -       "input": "0xa9059cbb0000000000000000000000004f641def1e7845caab95ac717c80416082430d0d000000000000000000000000000000000000000000000000000000000000002a",
            -       "maxFeePerGas": undefined,
            -       "maxPriorityFeePerGas": undefined,
            -       "to": "0xdAC17F958D2ee523a2206206994597C13D831ec7",
            -     },
            -   ],
            +   "method": "eth_gasPrice",
            +   "params": Array [],
              },
           3
              Object {
            -   "method": "eth_sendTransaction",
            -   "params": Array [
            -     Object {
            -       "data": "0xa9059cbb0000000000000000000000004f641def1e7845caab95ac717c80416082430d0d000000000000000000000000000000000000000000000000000000000000002a",
            -       "from": "0x8da5e39ec14b57fb9bcd9aa2b4500e909119795d",
            -       "gasPrice": "0x1ca14bd70",
            -       "input": "0xa9059cbb0000000000000000000000004f641def1e7845caab95ac717c80416082430d0d000000000000000000000000000000000000000000000000000000000000002a",
            -       "maxFeePerGas": undefined,
            -       "maxPriorityFeePerGas": undefined,
            -       "to": "0xdAC17F958D2ee523a2206206994597C13D831ec7",
            -     },
            -   ],
            +   "method": "eth_blockNumber",
            +   "params": Array [],
              },

    Number of calls: 8

       99 |                             amount,
      100 |                     );
    > 101 |                     expect(requestManagerSendSpy).toHaveBeenCalledWith({
          |                                                   ^
      102 |                             method: 'eth_sendTransaction',
      103 |                             params: [
      104 |                                     {

      at test/unit/contract_method_wrappers.test.ts:101:34
      at fulfilled (test/unit/contract_method_wrappers.test.ts:5:58)

Test Suites: 1 failed, 1 passed, 2 total
Tests:       2 failed, 4 passed, 6 total
```

### PR DESCRIPTION
**Pull request by Muhammad-Altabba**
## Description

Fixes #6133

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] I have selected the correct base branch.
- [ ] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] Any dependent changes have been merged and published in downstream modules.
- [ ] I ran `npm run lint` with success and extended the tests and types if necessary.
- [ ] I ran `npm run test:unit` with success.
- [ ] I ran `npm run test:coverage` and my test cases cover all the lines and branches of the added code.
- [ ] I ran `npm run build` and tested `dist/web3.min.js` in a browser.
- [ ] I have tested my code on the live network.
- [ ] I have checked the Deploy Preview and it looks correct.
- [ ] I have updated the `CHANGELOG.md` file in the root folder.
- [ ] I have linked Issue(s) with this PR in "Linked Issues" menu.

### CODE REVIEW

File: tools/web3-plugin-example/test/unit/contract_method_wrappers.test.ts [hunk 1]
```diff
@@ -69,15 +73,14 @@ describe('CustomRpcMethodsPlugin Tests', () => {
 				params: [
 					{
 						input: '0x70a082310000000000000000000000008da5e39ec14b57fb9bcd9aa2b4500e909119795d',
-						data: '0x70a082310000000000000000000000008da5e39ec14b57fb9bcd9aa2b4500e909119795d',
```
Comment (Spacesai1or) [inline tools/web3-plugin-example/test/unit/contract_method_wrappers.test.ts:72]: It [looks like](https://github.com/web3/web3.js/actions/runs/5134850870/jobs/9239561221?pr=6134#step:5:103) the removal of `data` needs to be reverted for this test and `transferAndGetBalances` test

**Comment by jdevcs** [comment]
some of unit tests are not passing,

File: tools/web3-plugin-example/test/unit/contract_method_wrappers.test.ts (hunk 1 above)
Comment (Muhammad-Altabba) [reply tools/web3-plugin-example/test/unit/contract_method_wrappers.test.ts:72]: This is interesting. I needed this change for the test to pass on my machine! I will investigate more on this. And if you have any clues, please let me know.

**Review by jdevcs — APPROVED** [review]

**Review by avkos — APPROVED** [review]

File: tools/web3-plugin-example/test/unit/contract_method_wrappers.test.ts (hunk 1 above)
Comment (Sammiegirl411) [reply tools/web3-plugin-example/test/unit/contract_method_wrappers.test.ts:72]: I'm trying to figure all this out

**Pull request merged**